web-animations-2/Overview.bs

```diff
@@ -138,10 +138,15 @@ urlPrefix: https://webidl.spec.whatwg.org/; type: dfn; spec: webidl
 urlPrefix: http://www.ecma-international.org/ecma-262/6.0/#sec-; type: dfn; spec: ecma-262
     text: code realms
     text: execution contexts
+urlPrefix: https://drafts.csswg.org/scroll-animations-1/; type: dfn; spec: scroll-animations-1
+    text: html-processing-model-event-loops
 </pre>
 
 <pre class=link-defaults>
 spec:webidl; type:dfn; text:throw
+spec:scroll-animations-1; type:dfn;
+    text:scroll-driven animation
+    text:html-processing-model-event-loop
```

Comment:
What are you trying to do here? “scroll-driven animation” should already be exported, and “html-processing-model-event-loop” is not a term, it's an anchor that you can reference via `[[scroll-animations-1#html-processing-model-event-loop|linking text here, or drop for heading substitution]]`

Reply:
Removed.